src/main/java/club/sk1er/hytilities/tweaker/asm/GuiPlayerTabOverlayTransformer.java

```diff
@@ -67,13 +67,16 @@ public void transform(ClassNode classNode, String name) {
                         if (next.getOpcode() == Opcodes.INVOKESTATIC) {
                             final String methodInsnName = mapMethodNameFromNode(next);
 
+                            InsnList list = new InsnList();
```

Comment:
move this inside of a new method, just to make the injection process look cleaner